src/common/components/CardFaces.js

```diff
@@ -76,4 +76,4 @@ const cardSet = [{
   }
 ];
 
-export const cardFaces = () => shuffleArray(doubleArray(cardSet));
+export const cardFaces = (numberOfCards = 5) => shuffleArray(doubleArray((shuffleArray(cardSet)).slice(0, numberOfCards)));
```

Comment:
I recommend renaming this function to something like `getShuffledCards`.

For readability, think of functions as performing actions or asking questions, i.e. verbs. By contrast, variables can be thought as nouns.

In this case, we want to get a list shuffled cards :)